README.md

```diff
@@ -8,38 +8,44 @@
 [![Build Status](https://circleci.com/gh/pyodide/pyodide.png)](https://circleci.com/gh/pyodide/pyodide)
 [![Documentation Status](https://readthedocs.org/projects/pyodide/badge/?version=latest)](https://pyodide.readthedocs.io/?badge=latest)
 
-Python with the scientific stack, compiled to WebAssembly.
+Python and the scientific stack, compiled to WebAssembly.
```

Comment:
The issue with "and" is that then less clear that it's Python scientific stack we are talking about not, the scientific stack in general.